css/jquery-glpi.css

```diff
@@ -262,6 +262,10 @@ html .ui-autocomplete {
 
 /* SELECT2 */
 
+.ui-widget {
+  font-family: Verdana,Arial,sans-serif;
```

Comment:
The old css custom includes a size change.

https://github.com/glpi-project/glpi/blob/558ccaf81db33e9259058b424f11b7fb26666f4c/lib/jquery/css/smoothness/jquery-ui-1.10.4.custom.css#L787

Reply:
As far as I cans ee, this is overriden by more specific rules; it should not change most of the cases (tabs)

Reply:
I tried it and did not really see a change.